src/Assets/Api.php

```diff
@@ -102,9 +136,9 @@ private function get_cached_script_data() {
 			return [];
 		}
 
-		$transient_value = json_decode( (string) get_transient( 'woocommerce_blocks_asset_api_script_data' ), true );
+		$transient_value = json_decode( (string) get_transient( $this->script_data_transient_key ), true );
```

Comment:
Should we add an extra check with `json_last_error_msg()` or use the `JSON_THROW_ON_ERROR` flag to be extra safe here? [Documentation](https://www.php.net/manual/en/function.json-decode.php)